communications/newsletters/newsletter_06_Sept2019.md

```diff
@@ -0,0 +1,70 @@
+Don't worry if you can't make it, there will be lots of opportunities for you to join us in future calls.
+Check out [the schedule](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md#dates-and-start-times) to add future dates to your calendar.
+
+## Funding approved
+
+We're delighted that *The Turing Way* has been awarded another round of funding from _Tools, Practices and Systems_ branch of the [AI for Science and Government programme](https://www.turing.ac.uk/research/ai-science-engineering-health-and-government) at the Turing.
+🎉 ✨ 💃 🕺 🚀 🌟 🙌
+
+You can read the application [in our GitHub repository](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/tps-funding-application-20190429.md) and we'll
+have more information in the coming months.
+We'd love for you to help shape the future of the project, so please [get in touch](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md#get-in-touch) with any comments or ideas.
+
+## New jobs
+
+Congratulations to three of the core _Turing Way_ team members who have started new jobs this summer.
+
+* [Rachael Ainsworth](https://twitter.com/rachaelevelyn) is now the Research Software Community Manager for the [Software Sustainability Institute](https://twitter.com/SoftwareSaved), based in Manchester.
+* [Patricia Herterich](https://twitter.com/PHerterich) has joined [Digital Curation](http://www.dcc.ac.uk/) to work on the [FAIRsFAIR](https://www.fairsfair.eu/) (Fostering Fair Data Practices in Europe) project in Edinburgh.
+* [Rosie Higman](https://twitter.com/RosieHLib) is the Research Data Manager for the [University of Sheffield Library](https://www.sheffield.ac.uk/library).
+
+We're so incredibly proud of all the Turing Way community members, and these roles in particular are incredibly important to the success of our goal of making "reproducible research too easy not to do".
+Congratulations Rachael, Patricia and Rosie!
+Well done 💖 😻 🌈
+
+## Artwork
+
+The beautiful artwork that Matt from Scriberia created at our May events is now available for your use and re-use on Zenodo: [https://doi.org/10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+
+Everything is [CC-BY licenced](https://creativecommons.org/licenses/by/4.0) and available in `jpg` and `svg` format.
+We'd love to know which ones you like the best, and to receive any pictures of them in action in your own presentations.
+
+If you adapt any of them (for example by changing the colours) please send us a copy so we can add it to the repository to make it available for others in the future.
+
+![](images/ComboScriberia_ForNewsletter.png)
+
+## Turing Way talks
+
+One of Kirstie's tasks now that its September is to update the GitHub repsitory with lots of the great talks and presentations that happened over the summer.
```

Comment:
```suggestion
One of Kirstie's tasks now that it's September is to update the GitHub repsitory with lots of the great talks and presentations that happened over the summer.
```

Reply:
Great catch - thank you. I fixed it locally (to explain why I didn't click "commit suggestion")